### PR TITLE
Remove duplicate definition

### PR DESCRIPTION
**Pull request by feihujiang**
Signed-off-by: feihujiang <jiangfeihu@huawei.com>

### CODE REVIEW

**Comment by feihujiang** [comment]
@rbuysse Could you please review this PR?

File: gossip/gossip_core.py [hunk 1]
```diff
@@ -145,7 +145,6 @@ def __init__(self, node, **kwargs):
 
         self.PendingAckMap = {}
         self.MessageHandledMap = {}
-        self.MessageHandlerMap = {}
```
Comment (boydjohnson) [inline gossip/gossip_core.py]: MessageHandlerMap and MessageHandledMap are two different dicts. The MessageHandlerMap is a map of message types to handler functions and the MessageHandledMap is a map of handled messages where keys are message identifiers and values are message expiration times. Checkout the code comments from lines 85-118 in gossip_core.py. Thanks for looking through the code.

**Comment by boydjohnson** [comment]
@feihujiang Checkout my comment on your code. This isn't a duplicate.

**Comment by feihujiang** [comment]
Yes, you are right!

**Pull request closed without merging**